source/postprocess/gravity_point_values.cc

```diff
@@ -300,35 +486,88 @@ namespace aspect
     void
     GravityPointValues<dim>::parse_parameters (ParameterHandler &prm)
     {
+      std::string gravity_subdirectory = this->get_output_directory() + "output_gravity/";
+      Utilities::create_directory (gravity_subdirectory,
+                                   this->get_mpi_communicator(),
+                                   true);
+
+      end_time = prm.get_double ("End time");
```

Comment:
Ah, the termination criteria manager can not return its plugins? Then you need to do this. But you should still be able to remove the declaration of this input parameter from the `declare_parameters` function above. It is sufficient to declare a parameter once.